commerce-service/src/main/java/com/liferay/commerce/internal/starter/CommerceRegionsStarterBase.java

```diff
@@ -0,0 +1,92 @@
+package com.liferay.commerce.internal.starter;
+
+import com.liferay.commerce.model.CommerceCountry;
+import com.liferay.commerce.service.CommerceCountryLocalService;
+import com.liferay.commerce.service.CommerceRegionLocalService;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.util.StringUtil;
+
+/**
+ * @author Riccardo Alberti
+ */
+public class CommerceRegionsStarterBase {
+
+	protected void start(
+			CommerceCountryLocalService commerceCountryLocalService,
+			CommerceRegionLocalService commerceRegionLocalService,
+			JSONFactory jsonFactory, ServiceContext serviceContext, int isoCode,
+			String layoutsPath)
+		throws Exception {
+
+		init(
+			commerceCountryLocalService, commerceRegionLocalService,
+			jsonFactory);
+
+		CommerceCountry commerceCountry = getCommerceCountry(
+			serviceContext.getCompanyId(), isoCode);
+
+		if (commerceCountry == null) {
+			return;
+		}
+
+		parseAndAddCommerceRegions(
+			commerceCountry, serviceContext, layoutsPath);
+	}
+
+	private CommerceCountry getCommerceCountry(long companyId, int isoCode)
```

Comment:
`private` method names should start with an underscore -- i.e. `_getCommerceCountry`. Same for the other private methods below.